diff --git a/helix-loader/src/lib.rs b/helix-loader/src/lib.rs
index a02a59af..84ed0f3d 100644
--- a/helix-loader/src/lib.rs
+++ b/helix-loader/src/lib.rs
@@ -41,13 +41,7 @@ pub fn runtime_dir() -> PathBuf {
         return conf_dir;
     }
 
-    // fallback to location of the executable being run
-    // canonicalize the path in case the executable is symlinked
-    std::env::current_exe()
-        .ok()
-        .and_then(|path| std::fs::canonicalize(path).ok())
-        .and_then(|path| path.parent().map(|path| path.to_path_buf().join(RT_DIR)))
-        .unwrap()
+    return PathBuf::from("/usr/lib64/helix").join(RT_DIR)
 }
 
 pub fn config_dir() -> PathBuf {
